guides/v2.3/rest/bulk-endpoints.md

```diff
@@ -9,6 +9,12 @@ functional_areas:
 
 Bulk API endpoints differ from other REST endpoints in that they combine multiple calls of the same type into an array and execute them as a single request. The endpoint handler splits the array into individual entities and writes them as separate messages to the message queue.
 
+{: .bs-callout .bs-callout-warning}
+By default, Magento uses RabbitMQ (AMQP) for processing message queues. Before using Bulk operations, you should see [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```

Comment:
Sorry, but I would disagree. I would align this note in a next way

```suggestion
Bulk API require to have RabbitMQ (AMQP) configured for processing message queues. Before using Bulk operations, you should see [RabbitMQ]({{ page.baseurl }}/install-gde/prereq/install-rabbitmq.html).
```

Reply:
Sorry maybe I was confusing.
So we are here talking about BULK API. And BULK API do not support DB for messages. Thats just not implemented at all and there are no option currently make it works, so we should not touch DB messages when we are talking about BULK or Async

Also this link https://devdocs.magento.com/guides/v2.3/extension-dev-guide/message-queues/implement-bulk.html#configmq is not really correct in this case. Cause again if we are talking about Bulk, this part with *xml configuration is already done but implementation itself.

So to run BULK you need only:
- Magento
- RabbitMQ configured and running
- Magento connected to use rabbitMQ (env.php file)
- Running consumer: 
* via CRON
* bin/magento command executed directly
* bin/magento command added and running as a service on environment with any comfortable for integrators way)

Reply:
Based on this conversation--Bulk API is configured to use RabbitMQ for message processing by default.  So customers should configure it to before using the Bulk API to process messages.

Reply:
Closing this conversation, which clarifies that Bulk API has its own default configuration for processing message queues, so you do not have to follow the process to configure message queues described in the Extension Guide.